third_party/accessibility/gfx/range/range_unittest.cc

```diff
@@ -4,16 +4,15 @@
 
 #include <sstream>
 
-#include "testing/gtest/include/gtest/gtest.h"
-#include "ui/gfx/range/range.h"
-#include "ui/gfx/range/range_f.h"
+#include "gtest/gtest.h"
+#include "range.h"
 
 namespace {
 
 template <typename T>
 class RangeTest : public testing::Test {};
 
-typedef testing::Types<gfx::Range, gfx::RangeF> RangeTypes;
+typedef testing::Types<gfx::Range> RangeTypes;
```

Comment:
We don't need RangeF? I thought they work side by side.

Reply:
Correct, we don't need `RangeF`. If we need it in future we can resurrect these unit tests but it's not needed for a11y on Windows.